operator/src/main/resources/scripts/introspectDomain.py

```diff
@@ -395,21 +395,6 @@ def getDynamicServersOrNone(self,cluster):
       ret = None
     return ret
 
-  # Work-around bugs in off-line WLST when accessing an SSL mbean
```

Comment:
I see you are refactoring this... We just fixed a bug in this area for CGBU where they weren't able to start a domain with the SSL MBean enabled. I want to make sure we don't regress this.

Reply:
It should be fine, I am familiar with the change as I mentioned to DongBo the probelm.